my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Get.java

```diff
@@ -5,23 +5,20 @@
 import java.nio.file.Paths;
 import java.nio.file.Path;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
 import java.io.ByteArrayOutputStream;
 
-
 public class Get implements RestMethod {
   private String status;
-  private String statusError = "HTTP/1.1 500 Internal Server Error";
   private String fileName;
-  private String CRLF ="\r\n";
-  String typeHeader = "Content-Type: ";
-  String contentLength = "Content-Length:";
   private String directory;
   private String contentType;
   private ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
- 
+  
+  private static final String STATUSERROR = "HTTP/1.1 500 Internal Server Error";
```

Comment:
Please follow Java's convention when declaring constants. http://www.java-tips.org/java-se-tips-100019/24-java-lang/802-how-do-i-declare-a-constant-in-java.html